README.md

```diff
@@ -89,6 +85,11 @@ $ go get github.com/glycerine/zygomys/cmd/zygo
 
 [See the wiki for lots of details and a full description of the zygomys language.](https://github.com/glycerine/zygomys/wiki).
 
+### Where did the name zygomys come from?
```

Comment:
This etymology and explanation of the name is very thoughtful and also fun, but I moved it down here so the very first things that a new user sees on the project page is how zygomys can help them solve problems, what zygomys is, and how it works.